README.md

```diff
@@ -2,8 +2,10 @@
 
 This package provides a network bridge which enables to exchange messages between ROS 1 and ROS 2.
 
-The bridge is currently implemented in C++ simply because there is no Python API for ROS 2 yet.
-Therefore it is limited to all the message types available at compile time.
+The bridge has currently only been implemented in C++ as at the time the Python API for ROS 2 had not been developed.
+Because of this its support is limited to only the message/service types available at compile time of the bridge.
+The bridge included with the prebuilt ROS 2 binaries supports the interface packages listed in the [ros2/common_interfaces repository.](https://github.com/ros2/common_interfaces)
```

Comment:
since `rcl_intefaces` don't have mappings associated with them they are not included in the packaged bridge. I agree that the statement should be made more general: https://github.com/ros2/ros1_bridge/commit/962c85bf369c45afdbaab8b5d7f4cc6ac0317ebe#diff-04c6e90faac2675aa89e2176d2eec7d8R7